### PR TITLE
Post qa fixes for posts improvements

### PR DESCRIPTION
**Pull request by Agill-Sheron**

### CODE REVIEW

File: src/modules/project/pages1/projectView/views/posts/views/PostCreateEdit.tsx [hunk 1]
```diff
@@ -67,6 +67,8 @@ export const PostCreateEdit = () => {
 
   const postForm = watch()
 
+  console.log('postForm', postForm.projectGoalIds)
```
Comment (steliosrammos) [inline src/modules/project/pages1/projectView/views/posts/views/PostCreateEdit.tsx:70]: console log

**Review by steliosrammos — COMMENTED** [review]
Dropped some comments

File: src/modules/project/pages1/projectView/views/posts/components/LinkGoalsAndRewardsModal.tsx [hunk 2]
```diff
@@ -157,29 +208,33 @@ const RewardItemSkeleton = () => {
...
-  const { inProgressGoals } = useGoalsAtom()
-
   return (
     <VStack w="full" alignItems="flex-start">
       <Body size="md" medium paddingX={standardPadding}>
         {t('Goals')}
       </Body>
       <VStack w="full" alignItems="start" paddingX={standardPadding} maxHeight={'200px'} overflowY={'auto'}>
-        {queryInProgressGoals.loading ? (
+        {queryInProgressGoals.loading || queryCompletedGoals.loading ? (
```
[the remark this answers is not in the document]
Comment (Agill-Sheron) [reply src/modules/project/pages1/projectView/views/posts/components/LinkGoalsAndRewardsModal.tsx:231]: They are two separate queries indeed

[the remark this answers is not in the document]
Comment (steliosrammos) [reply src/modules/project/pages1/projectView/views/posts/components/LinkGoalsAndRewardsModal.tsx:229]: Should it be two separate queries? The in progress goals and completed goals are resolved in a single query in the backend afair.

[the remark this answers is not in the document]
Comment (Agill-Sheron) [reply src/modules/project/pages1/projectView/views/posts/components/LinkGoalsAndRewardsModal.tsx:229]: iirc when I first implemented the goals I only used one query and then it was refactored in two separate queries in the re-design. I would wait on Sajal for any refactor there see maybe if there are good reasons to have them query distinctly

**Pull request closed without merging**